src/devices/DCMotor/DCMotorWithStartStop.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
The new header is only 2 lines vs 3 lines in the past

Reply:
corrected